config/webpack/environment.js

```diff
@@ -17,7 +17,6 @@ environment.plugins.prepend(
     clipboard: 'clipboard',
```

Comment:
Can you also remove the c3 and d3 lines from this file?  The move to highcharts removed those 2 dependencies, but I forgot to remove them at that time.